.github/workflows/ci.yml

```
@@ -120,6 +120,7 @@ jobs:
       - name: Test
         run: |
           cd legate-ci/github-ci/cunumeric
+          export PYTHONFAULTHANDLER=$(( ${{ matrix.name }} == "Eager"* ? 1 : 0 ))
```

```suggestion
          if [[ ${{ matrix.name }} == "Eager"* ]]; then export PYTHONFAULTHANDLER=1; fi
```
Python just checks if PYTHONFAULTHANDLER is non-empty, 0 and 1 both count as "enable"

---

Will fix that.